base/ltsockets.dtx

```diff
@@ -921,7 +986,23 @@
 }
 %    \end{macrocode}
 %  \end{macro}
-%  
+%
+%
+%
+% \begin{macro}[pTF, EXP] { \socket_if_plug_assigned:nn }
+%    Conditional testing the assignment of a plug.
+%    \changes{v0.9c}{2024/12/27}{Conditionals for sockets, plugs, and
+%                                assignments (gh/1577)}
+%    \begin{macrocode}
+\prg_new_conditional:Npnn \socket_if_plug_assigned:nn #1#2 { p , T , F , TF }
+  { \str_if_eq:vnTF { l__socket_#1_plug_str } {#2}
```

Comment:
Do we want to also expand the plug name (`#2`) here?

It's fully expanded in `\socket_if_plug_exist:nTF`.

Reply:
makes sense